packages/audio/alsa-utils/tmpfiles.d/alsa-utils.conf

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+d    /storage/.cache/alsa  0755 root root - -
+L    /var/lib/alsa/        -    -    -    - /storage/.cache/alsa
```

Comment:
Can do, but why?  This approach, besides being simple enough, has the added benefit of retaining the standard setup as much as possible.  A user familiar with ALSA on desktops for instance, needing to find the state file for whatever reason (say removing it, or trying to figure out why state is not retained), would know where to look for it.  This would not be the case, if we reconfigured `alsactl` to place it directly inside the non-standard `.cache/alsa` location.  Is there some other benefit to this approach?

Reply:
Have a look at prior art, e.g. openssh, where the configure option is applicable it is being used. There are  not many links from / into .cache `find . -xdev -type l | xargs ls -l | grep cache`